library/src/main/java/bio/terra/landingzone/stairway/flight/LandingZoneFlightMapKeys.java

```diff
@@ -8,6 +8,7 @@ public class LandingZoneFlightMapKeys {
   public static final String BILLING_PROFILE = "billingProfile";
   public static final String ATTACH = "attach";
   public static final String STORAGE_ACCOUNT_NAME = "storageAccountName";
+  public static final String REGION = "region";
```

Comment:
I'm not seeing where this gets used, can we remove it? Disregard if I'm just missing it

Reply:
I hadn't committed some changes from before the break.